lib/active_record_doctor/detectors/base.rb

```diff
@@ -150,6 +150,10 @@ def models(except: [])
       def underscored_name
         self.class.underscored_name
       end
+
+      def postgresql?
+        connection.adapter_name =~ /postg/i
```

Comment:
Let's use an array of adapter names that correspond to PostgreSQL. Less matching adapters, higher guarantee those that do match actually work.

Reply:
Changed.

Reply:
Thank you. I'll merge tomorrow.